behavior.c

```diff
@@ -371,33 +372,246 @@ static void behavior_leds_ntc(void ) {
 	}
 }
 
-static void pairing_tick(void) {
+
+
+static void setting_tick(void) {
+	#define SET_MENU (-1)
+	#define SET_VOLUME 0
+	#define SET_MOTOR 1
+	#define SET_RF_PARING 2
+	#define SPEED_STEP 128
+	#define BLINK 28
 	static unsigned char dbnc;
 	static unsigned char all_off;
-
-	if(buttons_state[0] | buttons_state[1] | 
-		buttons_state[3] | buttons_state[4] && all_off && 
-		(rf_get_status() & RF_PAIRING_MODE)) {
-		rf_pairing_stop();
-	}
+	static unsigned char setting_start;
+	static char setting_mode;
+	static char setting_select;
+	static char volume;
+	static char correction;
+	static unsigned char blink;
+	unsigned char ledrgb[3];
 
 	all_off = !(buttons_state[0] | buttons_state[1] | buttons_state[2]
-		| buttons_state[3] | buttons_state[4]);
-	
-	if(buttons_state[1] && buttons_state[4]) {
-		if(++dbnc > 100) {
-			if(dbnc == 101) {
-				if(!(rf_get_status() & RF_PAIRING_MODE)) {
+			| buttons_state[3] | buttons_state[4]);
```

Comment:
the diff becames strange when I supress the pairing tick because it used same piece of code...